lib/reply.js

```diff
@@ -47,6 +47,7 @@ function Reply (res, context, request, log) {
   this._serializer = null
   this._errorHandlerCalled = false
   this._isError = false
+  this._isRunningOnErroHook = false
```

Comment:
`_isRunningOnErroHook` you missing an `r` here in error which trickles down to all usages

Reply:
Nice catch!